apps/web/src/core/components/GetExtensionButton.tsx

```diff
@@ -0,0 +1,14 @@
+import clsx from "clsx"
+import { Button } from "~core/components/Button"
+import { EXTENSION_CHROME_URL } from "~core/components/common"
+
+export const GetExtensionButton = ({ className = "" }) => (
+  <Button
```

Comment:
small change here that was an issue with my original code, but it's likely that wrapping buttons in a next/link Link would be better for SEO? if you think so then let's do that for buttons like this going forward

Reply:
Yup - using a Link would be better for crawler to index the link.